scripts/workers/worker_util.py

```diff
@@ -12,6 +12,7 @@
     "get_challenge_by_queue_name": "/api/challenges/challenge/queues/{}/",
     "get_challenge_phase_by_pk": "/api/challenges/challenge/{}/challenge_phase/{}",
     "update_submission_data": "/api/jobs/challenge/{}/update_submission/",
+    "update_submission_status": "/api/jobs/challenge/{}/update_submission/",
```

Comment:
Hey, I missed this.
Can you remove this line as it is same as `update_submission_data`?

Reply:
@vkartik97 Okay